### PR TITLE
PR for Dynamic Port allocation #2

### PR DESCRIPTION
**Pull request by FRosner**
Issue: #2

### CODE REVIEW

**Comment by FRosner** [comment]
@pliguori can you review?

File: README.md [hunk 1]
```diff
@@ -4,16 +4,30 @@
...
+
+## Docker Container Name
+
+Using a named container will ensure that it gets removed properly the next time you start it again. You have to provide `NOMAD_DOCKER_CONTAINER_NAME` as an environment variable.
+
+## Environment Variable Replacement
+
+If you want to use environment variables inside your docker command that are passed by Nomad (e.g. dynamic ports), you can include them as a String and rely on the wrapper to replace them. To accomplish this, just set `NOMAD_DOCKER_ENVSUBST` to any value.
+
+*Please make sure that `envsubst` is installed on your system when using this option. It is part of the `gettext` package: `apt-get install gettext`.*
```
Comment (pliguori) [inline README.md:22]: How do we make sure about this in an automated scenario?

**Comment by pliguori** [comment]
LGTM. The above doubt is not really part of this wrapper, but of how we manage it

File: README.md (hunk 1 above)
Comment (FRosner) [inline README.md:22]: In your machine deployment (e.g. through ansible), you would install gettext alongside with docker and nomad.

**Pull request merged**